Demo/EarlGreyContribs/EarlGreyContribsSwiftTests/EarlGreyContribsSwiftTests.swift

```diff
@@ -18,13 +18,13 @@ import XCTest
 
 class EarlGreyContribsSwiftTests: XCTestCase {
   func testBasicViewController() {
-    EarlGrey().selectElementWithMatcher(grey_text("Basic View Controller"))
-        .performAction(grey_tap())
-    EarlGrey().selectElementWithMatcher(grey_accessibilityLabel("textField"))
-        .performAction(grey_typeText("Foo"))
-    EarlGrey().selectElementWithMatcher(grey_accessibilityLabel("showButton"))
-        .performAction(grey_tap())
-    EarlGrey().selectElementWithMatcher(grey_accessibilityLabel("textLabel"))
-        .assertWithMatcher(grey_text("Foo"))
+    EarlGrey().selectElement(with: grey_text("Basic View Controller"))
```

Comment:
as discussed, could we change EarlGrey() to EarlGrey by creating a class named EarlGrey and class-level methods like selectElement, rotate, etc.?